postgresql/database_test.go

```diff
@@ -57,32 +52,37 @@ var settings = ConnectionURL{
 	Password: password,
 }
 
-var host = flag.String("host", "testserver.local", "Testing server address.")
+var host = flag.String("host", "localhost", "Testing server address.")
```

Comment:
Sure, let's change that back. Perhaps we can support environment variables for this as well..? would be easier then passing flags in case when using a test runner.